kuma/wiki/kumascript.py

```diff
@@ -40,20 +38,26 @@ def should_use_rendered(doc, params, html=None):
             (force_macros or (not no_macros and not show_raw)))
 
 
-def post(request, content, locale=settings.LANGUAGE_CODE):
+def _post(content, env_vars, cache_control='',
```

Comment:
I would have used `cache_control=None` here, but strings are also immutable and using an empty string better indicates the type of what is expected.

Reply:
```suggestion
def _post(content, env_vars, cache_control=None,
```